projects/js-packages/social-logos/tools/build-react

```diff
@@ -11,8 +11,8 @@ cd "$root_dir"
 # Make dir if it doesn't exist.
 mkdir -p "$dest_react_dir"
 
-# Copy source files to build folder.
-cp "$src_react_dir/"* "$dest_react_dir"
+# Build files using TypeScript.
+pnpm tsc --noEmit false
```

Comment:
Why did you move the `--noEmit false` here instead of changing the tsconfig to extend `tsconfig.tsc.json` like the message told you to?

```suggestion
pnpm tsc
```